test/main/java/org/jetbrains/research/anticopypaster/ide/RefactoringNotifcationTaskTest.java

```diff
@@ -0,0 +1,16 @@
+package main.java.org.jetbrains.research.anticopypaster.ide;
+import org.jetbrains.research.anticopypaster.models.TensorflowModel;
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+
+
+public class RefactoringNotifcationTaskTest {
+    TensorflowModel model = new TensorflowModel();
```

Comment:
Did you manage to get it to build with this line uncommented? If so that's lit!

Reply:
Fixed with new commits